Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -66,11 +67,11 @@ const Checkbox = ({
 			sx={{
 				borderRadius: theme.shape.borderRadius,
 				p: theme.spacing(2.5),
```

Comment:
I think we are not supposed to use floats with MUI's spacing
https://mui.com/material-ui/customization/spacing/

Reply:
like mentioned in the chat, the left alignment issue of the notification email text and its previous child elements were already resolved, i'm not sure what we are trying to solve here

Reply:
This is something else. The point here is we are using spacing to have a rule for the whole project. If we pass floats here, we are bypassing that rule (the idea is that all spacing will be in a certain scale, in our case that is 2. When you pass .5, we are increasing space by 1)

Reply:
not sure why it was 2.5 historically, what we should be doing with this now or there will be another PR to address all floating spacing?